modules/drivers/bigquery-cloud-sdk/src/metabase/driver/bigquery_cloud_sdk.clj

```diff
@@ -843,7 +843,7 @@
   (let [details (get connection-details :details connection-details)
         client (database-details->client details)]
     (try
-      (for [query queries]
+      (doseq [query queries]
```

Comment:
This changes is either against the usual `driver/execute-raw-queries!` semantics, or, if we have changed the semantics, the ritual with `(or (and table-result (.getTotalRows table-result)) 0)` is superfluous. A `doall` or a `vec` around the `for` is needed in the first case.